lib/src/num_ext.dart

```diff
@@ -0,0 +1,87 @@
+/// Int extensions
+extension IntExt on int {
+  /// Returns ordinal of [int]
+  /// ``` dart
+  /// print(1.ordinal); // 1st
+  /// print(22.ordinal); // 22nd
+  /// print(143.ordinal); // 143rd
+  /// print(0.ordinal); // 0th
+  /// print((-1).ordinal); // -1st
+  /// print(12.ordinal); // 12th
+  /// print(69.ordinal); // 69th
+  /// ```
+  String get ordinal {
```

Comment:
The ordinal function here returns the English suffixes. Specify this in the comments, because it doesn't support localisation. 
For example the equivalent of `1st` in French is `1er`